app/services/discourse_reactions/reaction_notification.rb

```diff
@@ -18,7 +18,7 @@ def create
         return
       end
 
-      opts = { user_id: @user.id, display_username: @user.username }
+      opts = { user_id: @user.id, display_username: @user.username, display_name: @user.name }
```

Comment:
a general question for the entire PR: does it fallback to the username if `name` is empty?

Reply:
Just testing and it does _not_ 😭 thank you @CvX!